app/src/main/java/com/sun/chat_04/data/remote/UserRemoteDataSource.kt

```diff
@@ -0,0 +1,22 @@
+package com.sun.chat_04.data.remote
+
+import com.google.android.gms.tasks.OnFailureListener
+import com.google.android.gms.tasks.OnSuccessListener
+import com.google.firebase.database.FirebaseDatabase
+import com.sun.chat_04.data.model.User
+import com.sun.chat_04.util.Constants
+
+class UserRemoteDataSource(
+    private val user: User,
+    private val database: FirebaseDatabase,
+    private val onSuccessListener: OnSuccessListener<Void>,
+    private val onFailureListener: OnFailureListener
+) {
+
+    fun saveUserToRemote() {
```

Comment:
```suggestion
    fun saveUserToRemote(user: User, onSuccessListener: OnSuccessListener, onFailureListener: OnFailureListener) {
```
save user mà k thấy user đâu nhỉ?